src/test/resources/htsjdk/samtools/reference/leading_insertion.sam

```diff
@@ -0,0 +1,8 @@
+@HD	VN:1.0	SO:coordinate
+@SQ	SN:chrM	LN:16571	UR:file:/Users/mhanna/src/Sting/Homo_sapiens_assembly18.trimmed.fasta	M5:d2ed829b8a1628d16cbeee88e88e39eb
+@SQ	SN:chr20	LN:1000000	UR:file:/Users/mhanna/src/Sting/Homo_sapiens_assembly18.trimmed.fasta	M5:b4eac854d70893986ac578c53c2324f1
+@RG	ID:62A40.2	PL:illumina	PU:62A40AAXX101028.2	LB:Solexa-45345	DT:2010-10-28T00:00:00-0400	SM:RRBS885	CN:BI
+1	99	chrM	1	60	10I26M	chrM	1	0	AAAAAAAAAANNNNNNNNNNNNNNNNNNNNNNNNNN	AAAAAAAAAAAAAAABBBBBBBBBBBBBBBBBBBBB	RG:Z:62A40.2
```

Comment:
Would it be possible to use real bases here? E.g. the last 10bp of chrM concatenated with the first 26 (I assume that's what's causing these insertions?).